bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -213,7 +213,7 @@ public class ServerConfiguration extends AbstractConfiguration<ServerConfigurati
         "auditorMaxNumberOfConcurrentOpenLedgerOperations";
     protected static final String AUDITOR_ACQUIRE_CONCURRENT_OPEN_LEDGER_OPERATIONS_TIMEOUT_MSEC =
         "auditorAcquireConcurrentOpenLedgerOperationsTimeOutMSec";
-    protected static final String IN_FLIGHT_READ_ENTRY_NUM_IN_LEDGER_CHECKER = "inFlightReadEntryNumInLedgerChecker";
+    protected static final String READ_ENTRY_RATE_IN_LEDGER_CHECKER = "readEntryRateInLedgerChecker";
```

Comment:
The PR https://github.com/apache/bookkeeper/pull/2973 has been released in BookKeeper 4.15.0, we'd better mark the old one as deprecated instead of removing it. And set the old value to the new one if the user sets the old one.

Reply:
@hangc0276 I don't think the value of `inFlightReadEntryNumInLedgerChecker` should be set to `readEntryRateInLedgerChecker`, as the two settings have slightly different meanings. Therefore, I decided to restore the old setting. If both `readEntryRateInLedgerChecker` and `inFlightReadEntryNumInLedgerChecker` are set, `readEntryRateInLedgerChecker` takes precedence and `inFlightReadEntryNumInLedgerChecker` is ignored.